### ISSUE TITLE
Bug: deleted default items come back on server restart

### ISSUE THREAD

**Issue by josephclaytonhansen** [TomBursch/kitchenowl#611]
### Is this urgent?

Yes

### What parts are affected

Both

### What is the server version

106

### What is the client version

0.6.8

### What platform are you using

Android, Web, Mac, Windows

### What's the problem 🤔

When you delete things from the default items and the server restarts for any reason, all of them come back. This is a **huge pain**- it took nearly four hours to clean up the default item list, and I am deeply unhappy to find all of that work was wasted. Please look into this and fix it - I don't know if I can keep using KitchenOwl if I can't delete default items that are cluttering things up

**Comment by TomBursch** [comment]
Hi, thanks for the issue. Sorry you have a problem with the default items, but the idea is the more items the merrier and resulting in better search result. If you don't like them you can either set `SKIP_UPGRADE_DEFAULT_ITEMS` to true, which disables the addition of new (or removed) default items on startup (as described in the documentation) or just set the language to none on household creation which doesn't add any default items. Hope that helps.

**Comment by josephclaytonhansen** [comment]
Hey Tom,

Thanks for the reply. I don't have a problem with the existence of the default items, that's great. The problem is that deleting one of them doesn't actually delete it. This doesn't make any sense, deleting an item should delete it. As it stands, there's an immutable list of default items, and that's not good at all. The default items have a ton of duplicates (there's three different yeasts, for example) as well as dozens of uncategorized items that should be categorized. Changes to these default items are necessary as a user, and these changes should be permanently reflected in my database

**Comment by TomBursch** [comment]
That's true, but this only affects deletion. You can edit them as you like, and it's saved that they're not default anymore. Edited items are not automatically changed on server restart.
